### DISCUSSION TITLE
How can I get Arctic Code Vault Contributor?

### DISCUSSION THREAD

**Discussion by velocent** [Schweinepriester/github-profile-achievements#360] [Q&A] [answered]
Who knows?

**Comment by Schweinepriester** [comment]
You can't anymore, you had to have at least one commit in there:

<https://archiveprogram.github.com/arctic-vault/>:

> The 02/02/2020 snapshot archived in the GitHub Arctic Code Vault swept up every active public GitHub repository. It included every repo with any commits between the announcement at GitHub Universe on November 13th and 02/02/2020; every repo with at least 1 star and any commits from the year before the snapshot; and all repos with at least 250 stars. (It also included gh-pages for those repositories.) The snapshot consists of the HEAD of the default branch of each repository […]

**Comment by WHAHA-HA** [comment, marked as answer]
If you authored any commit on the default branch of a repository that was archived in the 2020 Arctic Vault program, you’ll get an Arctic Code Vault Contributor badge on your profile.
But you can't get this anymore. Try another achievements.

**Reply by riversdiniz** [reply]
Isso mesmo.
